src/test/java/divaeva/testNG/hw/RegistrationNewUserTest.java

```diff
@@ -0,0 +1,98 @@
+
+    private WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    public void openBrowser() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.navigate().to(TestUtils.ROZETKA_URL);
+    }
+
+    @AfterMethod(alwaysRun = true)
+    public void closeBrowser() {
+        driver.close();
+    }
+
+    private void fillInRegistrationInfo(String name, String surname, String phoneNumber, String email, String password) {
+        driver.findElement(REGISTRATION_BUTTON_ON_HOME_PAGE).click();
+        TestUtils.waitUntilElementVisibility(driver, REGISTER_BUTTON).click();
+
+        driver.findElement(NAME_FIELD).sendKeys(name);
+        driver.findElement(SURNAME_FIELD).sendKeys(surname);
+        driver.findElement(PHONE_NUMBER_FIELD).sendKeys(phoneNumber);
+        driver.findElement(EMAIL_FIELD).sendKeys(email);
+        driver.findElement(PASSWORD_FIELD).sendKeys(password);
+        driver.findElement(REGISTER_BUTTON).click();
+    }
+
+    @Test(groups = "positive")
+    public void registrationWithCyrillicFilledFields() {
+        fillInRegistrationInfo("Лера", "Секач", PHONE_NUMBER_FIELD_VALUE, EMAIL_FIELD_VALUE, PASSWORD_FIELD_VALUE);
+
+        WebElement registrationConfirmationPage = driver.findElement(REGISTRATION_CONFIRMATION_PAGE);
+        Assert.assertTrue(registrationConfirmationPage.isDisplayed(), "Registration was successful.");
+    }
+
+    @Test(groups = "negative")
+    public void registrationWithLatinFilledFields() {
+        fillInRegistrationInfo("Lera", "Sekach", PHONE_NUMBER_FIELD_VALUE, EMAIL_FIELD_VALUE, PASSWORD_FIELD_VALUE);
+        String expectedNameFieldError = "Введіть своє ім'я кирилицею";
+        String expectedLastNameFieldError = "Введіть своє прізвище кирилицею";
+
+        List<WebElement> validationErrorMessages = TestUtils.waitUntilElementsPresence(driver, VALIDATION_ERROR_MESSAGES);
+
+        boolean isNameErrorFound = false;
+        boolean isLastNameErrorFound = false;
+        for (WebElement validationErrorMessage : validationErrorMessages) {
+            if (validationErrorMessage.getText().equals(expectedNameFieldError)) {
+                isNameErrorFound = true;
+            }
+            if (validationErrorMessage.getText().equals(expectedLastNameFieldError)) {
+                isLastNameErrorFound = true;
+            }
+        }
+
+        Assert.assertTrue(isNameErrorFound, "Name field's validation error message is not displayed");
+        Assert.assertTrue(isLastNameErrorFound, "Last Name field's validation error message is not displayed");
+    }
+
+    @Test(groups = "negative")
+    public void registrationWithEmptyFields() {
+        driver.findElement(REGISTRATION_BUTTON_ON_HOME_PAGE).click();
+        TestUtils.waitUntilElementVisibility(driver, REGISTER_BUTTON).click();
+        driver.findElement(REGISTER_BUTTON).click();
+
+        List<WebElement> emptyFormFields = driver.findElements(EMPTY_REGISTRATION_FORM_FIELDS);
```

Comment:
As far as I understood this xpath "EMPTY_REGISTRATION_FORM_FIELDS" should return empty fields, but in fact it returns all fields even if they aren't empty.
So, maybe test should be slightlt changed?

Reply:
Changed naming and Assert